tests/components/pi_hole/test_init.py

```diff
@@ -126,24 +111,20 @@ async def test_disable_service_call(hass):
 
         await hass.async_block_till_done()
 
-        assert mock_disable.call_count == 2
+        assert mocked_hole.disable.call_count == 2
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)